src/workerd/util/tracing.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <kj/memory.h>
+#include <kj/string.h>
+#define PERFETTO_ENABLE_LEGACY_TRACE_EVENTS 1
+#include "perfetto/tracing.h"
+
+PERFETTO_DEFINE_CATEGORIES_IN_NAMESPACE(
+    workerd,
```

Comment:
I don't feel strongly about this at all, but wanted to mention I ended up going with the `edgeworker::perfetto` namespace internally. The (perhaps major) downside is that it means all `perfetto` namespaces need to become `::perfetto`, so I'm not sure it was smart. I'll align with what you think.

Reply:
hmm, not sure. Maybe something like `{...}::trace_events` ?

Reply:
For workerd, I'm gonna go with `workerd::traces` as the namespace.